themes/vue/layout/partials/main_menu.ejs

```diff
@@ -10,3 +10,7 @@
 </li>
 <%- partial('partials/support_vue_dropdown') %>
 <%- partial('partials/language_dropdown') %>
+<a class="button nav-button gray has-icon" href="https://github.com/vuejs/vue" target="_blank">
```

Comment:
As this a direct child of `<ul>`, I believe it should be wrapped inside a `<li>`.

Reply:
Thank you for spotting it 😅 pushed a visual fix (I still don't like how it looks though, suggestions welcome!)